py/code_intelligence/triage.py

```diff
@@ -69,7 +69,9 @@ def from_issue(cls, issue):
         break
 
     # TODO(jlewi): Could we potentially miss some events since we aren't
-    # paginating through all events for an issue?
+    # paginating through all events for an issue? This should no longer
```

Comment:
@jlewi this file has moved to `py/issue_triage/`

Reply:
Done.